envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -3,6 +3,16 @@ stack_env: example_ci-ceph_swift
 
 state_path_base: /opt/stack/data
 
+swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
+swift_undercloud_floating_ip: "{{ swift_floating_ip }}"
+swift_fqdn: "swift.{{ fqdn }}"
+
+etc_hosts:
+  - name: "{{ fqdn }}"
+    ip: "{{ undercloud_floating_ip }}"
+  - name: "{{ swift_fqdn }}"
+    ip: "{{ swift_undercloud_floating_ip }}"
```

Comment:
As far as I can see, this is fine for ursula because the fqdn entry that it would have overwritten is duplicated here.